core/arch/arm/plat-imx/drivers/imx_csu.c

```diff
@@ -1,135 +1,239 @@
 // SPDX-License-Identifier: BSD-2-Clause
 /*
- * Copyright 2017-2019 NXP
- *
+ * Copyright 2021 Foundries.io
```

Comment:
how is so? it is not the same driver - 80% of the original file has gone and the other 20% are register settings. so it is a new file.

Reply:
I don't think it works like that. It's still the same file. Whatever the amount of lines left from the previous change, you cannot just remove a copyright from a file. It must be a new file.

Reply:
And of course I agree, it is a legal issue removing copyrights. However I am not removing any copyrights since this is a new file. why do you think it is the same file? based on what? the register settings?